### PR TITLE
Update Larren's & Brimstone openable

### PR DESCRIPTION
**Pull request by nwjgit**
### Description:
Update Larren's & Brimstone openable to allow for fish rolls.
### Changes:
- Update Larran's openable
- Update Brimstone openable
### Other checks:
- [X] I have tested all my changes thoroughly.
- Supports: https://github.com/oldschoolgg/oldschoolbot/pull/5426
- This PR will still work without merging: https://github.com/oldschoolgg/oldschooljs/pull/357 but it will only roll one type of fish. Once https://github.com/oldschoolgg/oldschooljs/pull/357 is merged it will work similar to ingame.

### CODE REVIEW

File: src/lib/openables.ts [hunk 1]
```diff
@@ -162,7 +165,20 @@ const osjsOpenables: UnifiedOpenable[] = [
...
+		output: async (
+			args: OpenArgs
+		): Promise<{
+			bank: Bank;
+		}> => {
+			const chest = new BrimstoneChestOpenable(BrimstoneChest);
+			const fishLvl = args.user.skillLevel(SkillsEnum.Fishing);
+			const brimstoneOptions: any = {
+				fishLvl
+			};
```
Comment (gc) [inline src/lib/openables.ts:177]: This `any` shouldnt be needed

Comment (nwjgit) [reply src/lib/openables.ts:177]: Fixed

File: src/lib/openables.ts [hunk 2]
```diff
@@ -234,7 +250,21 @@ const osjsOpenables: UnifiedOpenable[] = [
...
 		],
-		output: Openables.LarransChest.table,
+		output: async (
+			args: OpenArgs
+		): Promise<{
+			bank: Bank;
+		}> => {
+			const chest = new LarransChestOpenable(LarransChest);
+			const fishLvl = args.user.skillLevel(SkillsEnum.Fishing);
+			const larransOptions: any = {
```
Comment (gc) [inline src/lib/openables.ts:260]: any here too

Comment (nwjgit) [reply src/lib/openables.ts:260]: This one errors out without the any. I've tried a few things but couldn't get it to work without the any

Comment (nwjgit) [reply src/lib/openables.ts]: nvm i got it working.

**Pull request merged**